Makefile

```
@@ -6,8 +6,13 @@ TARGET_DIR=bin/
 
 .PHONY: build test clean
 
+fmt:
+	goimports -w cmd/*/*.go pkg/*/*.go
```

goimportsは指定ディレクトリ以下も掘ってフォーマットかけてくれるので，
明示的にディレクトリを指定しなくても良いと思います．

```suggestion
	goimports -w .
```